src/components/Errors/ErrorsValidationForm/ErrorsValidationForm.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {TextComponent} from 'src/styled-components/globalStyles/GlobalStyles.styled';
+
+interface ErrorsValForm {
+  [key: string]: any;
+  errorKey: string;
+}
+
+export const ErrorsValidationForm = ({
+  errors,
+  errorKey,
+}: ErrorsValForm): JSX.Element => {
+  return errors[errorKey] ? (
+    <TextComponent fontSizeType="text" fontWeight="bold" color="red">
```

Comment:
use theme in all the cases where you use a color

Reply:
I'm using theme in TextComponent writed in GlobalStyles.styled.ts
```
const TextComponent = styled.Text<FontProps>`
  font-size: ${({fontSizeType, theme}: FontProps) =>
    theme.fontSizeType[fontSizeType]};
  font-weight: ${({fontWeight, theme}: FontProps) =>
    theme.fontWeights[fontWeight]};
  color: ${({color, theme}: FontProps) => theme.colors[color]};
  text-align: ${props => props.textAlign || 'center'};
`;
```

Reply:
ok